tamr_unify_client/client.py

```diff
@@ -102,7 +103,11 @@ def request(self, method: str, endpoint: str, **kwargs) -> requests.Response:
 
         # Attempt request with auth cookie
         response = self.session.request(method, url, **kwargs)
-        if response.status_code == 401 and "credentials" in response.text.lower():
+        if (
+            response.status_code == 401
+            and "credentials" in response.text.lower()
+            and not isinstance(self.session.auth, JwtTokenAuth)
```

Comment:
if `set_auth_cookie()` only works for `UsernamePasswordAuth`, then it would make more sense to check here that you are using `UsernamePasswordAuth`, instead of the negative check for `JwtTokenAuth`